lib/ws-kernel-picker.js

```diff
@@ -91,7 +97,9 @@ export default class WSKernelPicker {
           this.onSession.bind(this)
         );
         sessionListing.previouslyFocusedElement = this.previouslyFocusedElement;
-        sessionListing.setLoading("Loading sessions...");
+        sessionListing.selectListView.update({
+          loadingMessage: "Loading sessions..."
+        });
```

Comment:
I'd set the breakpoint on [this line](https://github.com/BenRussert/hydrogen/blob/66fa4244b86d07c32c399e7cce24a430651c3bff/lib/ws-kernel-picker.js#L129) and then *step in* (press the button with an arrow pointing down, or F11 in linux).